dev-docs/dev.md

```diff
@@ -824,6 +825,7 @@ Buffer files - Fluent Bit
 Description                               | Link
 ----------------------------------------- | ----
 Config file - Ops Agent                   | `C:\Program Files\Google\Cloud Operations\Ops Agent\config\config.yaml`
+Logs - Health Checks                      | `C:\ProgramData\Google\Cloud Operations\Ops Agent\log\heath-checks.log`
```

Comment:
s/heath/health/

Reply:
Good catch! This also makes me realize that the public docs have the same typo.

Reply:
Fixed